internal/renew_root.go

```diff
@@ -75,3 +93,102 @@ func (s fetchRootStep) Execute(h *rootRenewalContext) error {
 	}
 	return nil
 }
+
+func loadCertsFromPem(data []byte) (certs []*x509.Certificate, err error) {
+	var block *pem.Block
+	if len(data) == 0 {
+		return nil, errors.New("Unexpected empty PEM block")
+	}
+	for len(data) > 0 {
+		if block, data = pem.Decode(data); block == nil {
+			return nil, fmt.Errorf("Malformed PEM block at %s", data[:100])
```

Comment:
maybe put some `...` in the error message to indicate the string has been truncated for display purposes?